test/unit/basic_test.rb

```diff
@@ -70,7 +70,7 @@ def test_client_logs_without_response
       %i[post put patch].each do |method|
         stub = stub_request(method, /basic-test/).to_timeout
         assert_raise(RestClient::RequestTimeout, RestClient::Exceptions::OpenTimeout) do
-          client.send(method, stubbed_path, FHIR::Patient.new, format_headers)
+          client.send(method, stubbed_path, FHIR::STU3::Patient.new, format_headers)
```

Comment:
It seems like there are a lot of tests that were altered so `FHIR::` just becomes `FHIR::STU3::`...

I mean, that is one way to fix them.

Another way would be to upgrade the unit tests so they actually just test R4... which seems like what you really want to do here, since R4 is the first normative version of FHIR, and most vendors seem to be preparing to skip over STU3.

Reply:
That makes sense to me, I'll take a look

Reply:
I altered the tests to use the FHIR::Models instead of FHIR::STU3::Models, except for multi-version tests.  In those cases I added R4 tests in addition to the existing STU3 and DSTU2 tests.